### ISSUE TITLE
no-raw-text

### ISSUE THREAD

**Issue by BPetronel** [Intellicode/eslint-plugin-react-native#271]
On "eslint-plugin-react-native": "3.9.1" there seems to be a false positive error/warning
```
<Text testID="OuterTestId">
	Message from
	<Text numberOfLines={1} ellipsizeMode="middle" style={styles.middleText}>
		{username}
	</Text>
</Text>
```
There seems to be a problem with Nested Text for this rule.

**Comment by rsml** [comment]
The issue exists on v3.9.0 too. v3.8.1 is fine though. I'm testing with a custom text component like so:
`"react-native/no-raw-text": [2, {
      "skip": [
        "Label",
      ]
    }],`

**Comment by Intellicode** [comment]
Any help on this issue is appreciated, I think its the same as this: https://github.com/Intellicode/eslint-plugin-react-native/issues/269

**Comment by lsps9150414** [comment]
same even with eslint: v3.81

**Comment by DaniyarJakupov** [comment]
Still present in `"eslint-plugin-react-native": "^3.10.0"`

**Comment by ejain** [comment]
Still present in 3.11.0. As a workaround, you can configure the rule with `{ "skip": [ "Text.Text" ] }]` (add `"Text.Text.Text"` for two levels of nesting etc).

**Comment by leotm** [comment]
Noticed in 3.11.0 too, cheers @ejain this works for me

```ts
'react-native/no-raw-text': ['error', { skip: ['Text.Text'] }]
```

**Comment by pipedreambomb** [comment]
Yeah, I found this really confusing as nesting text like this is [exactly what the React Native docs tell you to do](https://reactnative.dev/docs/text).
